Example/Podfile.lock

```diff
@@ -21,8 +23,8 @@ EXTERNAL SOURCES:
     :path: "../"
 
 SPEC CHECKSUMS:
-  Swiftilities: 31a6a2e6d4eaae417c572b492a74e68886ca4c86
+  Swiftilities: f47d7265bfd744240d90d90f85056908c8979fed
 
 PODFILE CHECKSUM: b1d3049f81e75c3304f48a3fb9386cc1458c9549
 
-COCOAPODS: 1.0.0
+COCOAPODS: 1.0.1
```

Comment:
#req please run `bundle exec pod install` instead of just `pod install` so that you don't change the CocoaPods version. We can do that in a separate PR.